lab-sam/lib/greeting.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const greeter = {};
+
+greeter.greeting = function(name) {
```

Comment:
This function should really be split into two files

their should be a main program in the root of your project called index.js or hello.js
and that program should import a this greeting.js module

this modules should only take in a name and output 'hello' + name 
* process.argv should probbly be parsed in the main program then passed into this module as a paramter
* it should not console.log()